tests/functional/pv/pv_services/test_cephfs_fsync_consistency.py

```diff
@@ -0,0 +1,91 @@
+import logging
+
+from ocs_ci.framework.testlib import ManageTest, tier2, bugzilla, green_squad
+from ocs_ci.helpers import helpers
+from ocs_ci.ocs import constants
+from ocs_ci.ocs.node import get_worker_nodes
+
+
+logger = logging.getLogger(__name__)
+
+
+@tier2
+@bugzilla("2212310")
+@green_squad
+class TestCephfsFsyncConsistency(ManageTest):
+    """
+    Ensuring File Completeness Post-fsync with Shared CephFS Volume
+
+    """
+
+    def test_cephfs_fsync_consistency(self, teardown_project_factory):
```

Comment:
There was a suggestion to enhance the existing test,test_rbd_rwx_pvc.py tests. Did you check on it?

Adding the email discussion below,
```
Would it work to enhance [test_rbd_rwx_pvc.py](https://github.com/red-hat-storage/ocs-ci/blame/master/tests/functional/pv/pv_services/test_rbd_rwx_pvc.py) with a server reader pod?
[+Jilju Joy](mailto:jijoy@redhat.com) to share his thoughts 
I think that will be possible with a  different way(referring to the PR) to write from the client and read. Currently the test case test_rbd_rwx_pvc.py does write from one node(pod on one node) and verifies md5sum from that pod as well as other pods on different worker nodes.
```